src/mlpack/methods/linear_regression/linear_regression.cpp

```diff
@@ -141,7 +141,7 @@ double LinearRegression::ComputeError(const arma::mat& predictors,
       Log::Fatal << "The test data must have the same number of columns as the "
           "training file." << std::endl;
     }
-    temp = responses - arma::trans(parameters) * predictors;
+    temp = responses - (arma::trans(parameters) * predictors);
```

Comment:
I think I already commented on this one, I don't think we have to add ``()`` around a multiplcation. Also I think you could have incorporated this one into: https://github.com/mlpack/mlpack/pull/1789, just to keep the number of open PR's overseeable.

Reply:
@zoq , @micyril  please review my changes in f1_impl.hpp

Reply:
I don't see a need for this particular change either.